packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,52 @@ class FancyFocus extends FocusManager {
 
 ### Properties
 
-| name          | type                                                    | readonly | description                                                                                                                  |
-| ------------- | ------------------------------------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| direction     | 'column'\|'row'                                         | false    | determines whether focus handling responds to left/right or up/down key                                                      |
-| selected      | <DocsLink id="lng.Component" />                         | true     | current item in focus                                                                                                        |
```

Comment:
Similarly, `selected` only has a getter which makes the prop read-only so this was correctly documented but is it necessary in the props table? @erautenberg